tests/mod.rs

```diff
@@ -26,6 +26,8 @@ fn run_test(name: &str) -> Result<(), cobalt::Error> {
         .unwrap_or_default();
     let destdir = TempDir::new(name).expect("Tempdir not created");
 
+    println!("Test Config {}: {:?}", name, config);
```

Comment:
There a reason to print what the config looks like?  Its on disk for someone to look at in case of failure.

Reply:
Ehem. That shouldn't be there...